sonar-web-plugin/src/main/java/org/sonar/plugins/web/checks/header/HeaderCheck.java

```diff
@@ -44,30 +47,75 @@
   @RuleProperty(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder annotation arguments to match the order of declaration. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS3340)